VMBackup/main/Utils/EventLoggerUtil.py

```diff
@@ -245,6 +245,13 @@ def _send_event_file_to_event_directory(self, file_path, events_folder):
             FileHelpers.deleteFile(file_path)
             print("Information: Event reporting has paused due to reaching maximum capacity in the Event directory. Reporting will resume once space is available. Events for this iteration will not be reported.")
 
+    def clear_temp_directory(self, directory_path):
+        try:
+            if os.path.exists(directory_path):
+                os.rmdir(directory_path)
```

Comment:
[os.rmdir](https://docs.python.org/3/library/os.html#os.rmdir) only works if the folder does not have any subfolders or files inside it - basically when it's empty. It will throw an exception. Use it if and only if you are sure that the directory is empty. Otherwise please look at something like [shutil.rmtree](https://docs.python.org/3/library/shutil.html#shutil.rmtree).

Reply:
So even the intention behind using os.rmdir was to clear an empty directory as this method would be called only at the end where all the files from temp directory would be moved to event folder and then we would be clearing out the temp folder. 
But yes it would be better to have that check, will update it.